first

```diff
@@ -2,9 +2,5 @@
 
 {config: input=1, output=2}
 
-{Feature: first-feature}
+{Feature: first}
```

Comment:
Add another line to set priority

{Priority: moderate}

Reply:
Tested the change. Works fine

Reply:
add the additional line also.